src/dhcp/aca_dhcp_server.cpp

```diff
@@ -608,18 +608,17 @@ ACA_Dhcp_Server::_pack_dhcp_offer(dhcp_message *dhcpdiscover, dhcp_entry_data *p
   return dhcpoffer;
 }
 
-void ACA_Dhcp_Server::_parse_dhcp_request(dhcp_message *dhcpmsg)
+void ACA_Dhcp_Server::_parse_dhcp_request(uint32_t in_port, dhcp_message *dhcpmsg)
 {
   string mac_address;
   dhcp_entry_data *pData = nullptr;
   dhcp_message *dhcpack = nullptr;
   dhcp_message *dhcpnak = nullptr;
   struct sockaddr_in sa;
-  uint32_t self_sid = 0;
+  uint32_t self_sid = 2130706433;
```

Comment:
it will be good to comment what this id map to (an ip address we defined?). Let's put this as a static define on top of this file so that we can easily identify and change it in the future if needed.